### PR TITLE
Refactor byte translation

### PR DESCRIPTION
**Pull request by pulquero**
Reduce code duplication and ensure single point of responsibility for Value <-> byte translation.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>findbugs-maven-plugin</artifactId>
-                <version>3.0.0</version>
+                <version>3.0.4</version>
```
Comment (pulquero) [inline pom.xml]: Upgraded to work with latest Maven.

File: tools/src/main/java/com/msd/gin/halyard/tools/HalyardElasticIndexer.java [hunk 2]
```diff
@@ -74,7 +76,7 @@
 
         final SimpleValueFactory ssf = SimpleValueFactory.getInstance();
         long counter = 0, exports = 0, batches = 0, statements = 0;
-        byte[] lastHash = new byte[20], hash = new byte[20];
+		byte[] lastHash = new byte[HalyardTableUtils.KEY_SIZE], hash = new byte[HalyardTableUtils.KEY_SIZE];
```
Comment (pulquero) [inline tools/src/main/java/com/msd/gin/halyard/tools/HalyardElasticIndexer.java]: Missing use of KEY_SIZE constant.

File: tools/src/main/java/com/msd/gin/halyard/tools/HalyardStats.java [hunk 3]
```diff
@@ -150,20 +150,16 @@ private boolean matchingGraphContext(Resource subject) {
...
                 if (!matchAndCopyKey(key.get(), key.getOffset() + 1, lastCtxFragment) || region != lastRegion) {
                     cleanup(output);
-                    Cell c[] = value.rawCells();
-                    ByteBuffer bb = ByteBuffer.wrap(c[0].getQualifierArray(), c[0].getQualifierOffset(), c[0].getQualifierLength());
-                    int skip = bb.getInt() + bb.getInt() + bb.getInt();
-                    bb.position(bb.position() + skip);
-                    byte[] cb = new byte[bb.remaining()];
-                    bb.get(cb);
-                    graph = NTriplesUtil.parseURI(new String(cb,UTF8), ssf);
+					stmts = HalyardTableUtils.parseStatements(value, ssf);
```
Comment (pulquero) [inline tools/src/main/java/com/msd/gin/halyard/tools/HalyardStats.java]: Re-use parseStatements() rather than duplicate byte translation code.

**Comment by asotona** [comment]
Nice job, Thanks!

**Pull request merged**